app/views/subscription/bookmarklet.phtml

```diff
@@ -15,4 +15,4 @@
 	<h2><?= _t('sub.api.title') ?></h2>
 	<p><?= _t('sub.api.documentation') ?></p>
 	<kbd><?= Minz_Url::display(array('c' => 'feed', 'a' => 'add'), 'html', true) ?>&amp;url_rss=%s</kbd>
-</div>
\ No newline at end of file
+</main>
```

Comment:
I think we prefer newline at end of file, or is that just me? In any case I prefer not seeing git/diff talking about it. ;-) (Lesser nit.)

Reply:
New line at the end of the file indeed, and that should be automatic in editors supporting https://github.com/FreshRSS/FreshRSS/blob/edge/.editorconfig

Reply:
added the newline.
(it was not before there)